source/developers/how-tos/enable-translations-new-repo.rst

```diff
@@ -5,21 +5,196 @@ Enabling Translations on a New Repo
+
+   Add a new entry under the ``python-translations`` section. For example for the XBlock located in
+   ``https://github.com/openedx/MyXBlock`` and has the XBlock Python module name ``my_ai_bot_xblock`` it should have
+   the following entry::
+
+    python-translations:
+      strategy:
+        matrix:
+          repo:
+            ...
+            - repo_name: MyAIBotXBlock
+              python_module_name: my_ai_bot_xblock
+            ...
+
+   When adding a theme or any other module that's not an Open edX plugin or an XBlock the entry should
+   added without ``python_module_name`` as demonstrated below::
+
+    python-translations:
+      strategy:
+        matrix:
+          repo:
+            ...
+            - repo_name: credentials-themes
+            ...
+
+5. Create a pull request in the `openedx-translations repo`_ and wait for review and approval.
+   Once the pull request is merged the GitHub/Transifex workflow will be ongoing as described in the
+   Introduction section. No additional configuration is needed.
+
+6. Verify the workflow is syncing the translations properly as described in the Debugging section below.
+
+7. Install the XBlock or plugin in a devstack and run the ``OPENEDX_ATLAS_PULL=true make pull_translations`` in
+   the edx-platform repo to preview the translations in devstack.
+
+   This step is somewhat oversimplified and assumes you're familiar with the Open edX development workflow whether
+   you're using `Tutor <https://docs.tutor.overhang.io/>`_ or the
+   `Open edX devstack <https://github.com/openedx/devstack/>`_.
+
+================================
+Django Microservice Repos (IDAs)
+================================
+
+In the context of Open edX both of the Microservice and Independently Deployable Application (IDA) terms are used
+interchangeably. There are many Django microservices in the Open edX ecosystem.
+The most common ones are: ``edx-platform`` itself, ``credentials``, ``ecommerce`` and ``course-discovery``.
+
+1. If you are not working within an existing repository, create a new repo using the `cookiecutter-django-ida`_
+   template. If the repo already exists continue to the next steps.
+
+TODO: How to enable translations infrastructure on a Django microservice
+
+5. Create a pull request in the `openedx-translations repo`_ and wait for review and approval.
+   Once the pull request is merged the GitHub/Transifex workflow will be ongoing as described in the
+   Introduction section. No additional configuration is needed.
+
+6. Verify the workflow is syncing the translations properly as described in the Debugging section below.
+
+7. Run the ``OPENEDX_ATLAS_PULL=true make pull_translations`` to verify that it pulls the translations from the
+   `openedx-translations repo`_ into ``conf/locale`` directory. Depending on the repo
+   ``make static``/``make static.dev`` is usually needed to generate the JavaScript translation files.
```

Comment:
does it make sense to move this step above the PR step? it seems like testing locally is something that should happen before PRing, but the listed command won't work pre-PR